pkg/cmd/server/api/validation/validation.go

```diff
@@ -55,6 +55,12 @@ func ValidateServingInfo(info api.ServingInfo) fielderrors.ValidationErrorList {
 	allErrs = append(allErrs, ValidateHostPort(info.BindAddress, "bindAddress")...)
 	allErrs = append(allErrs, ValidateCertInfo(info.ServerCert, false)...)
 
+	switch info.BindNetwork {
+	case "tcp", "tcp4", "tcp6":
+	default:
+		allErrs = append(allErrs, fielderrors.NewFieldInvalid("bindNetwork", info.BindNetwork, "must be 'tcp', 'tcp4', or 'tcp6'"))
```

Comment:
you can bind on tcp, tcp4, and tcp6 by providing 0.0.0.0.  All ipv4 addresses can be converted to special ipv6 addresses.